### DISCUSSION TITLE
WIN then TYPE-TO-SEARCH randomly stops working

### DISCUSSION THREAD

**Discussion by DavidBevi** [valinet/ExplorerPatcher#2532] [Q&A]
I'm used to **press the Win button and just start typing to search** files and folders. **Sometimes it doesn't work**. I've tried relaunching explorer, but it does nothing.

Can you tell me if there's a solution?

**EP Ver**: 22621.2506.60.1 -- **Win Ver**: 11 Pro 22621.2715 -- **Taskbar style**: Win10 -- **Start menu style**: Win10
(I don't know which info is relevant and which not, if asked I'll gladly edit it!)

**Comment by pyrates999** [comment]
When it happens, try to open up task manager and kill explorer.exe and then run task manager again and run explorer.exe

**Reply by DavidBevi** [reply]
OK. I already tried re-launching Explorer from **Processes** section (it didn't resolve the issue), so I'll try killing explorer.exe from **Details** section as you suggest and launching it again.
